centreon/src/Centreon/Domain/Repository/Interfaces/DataStorageEngineInterface.php

```diff
@@ -33,19 +33,25 @@ interface DataStorageEngineInterface
      * Rollback the operations in the transaction
      *
      * @return bool
+     *
+     * @throws \Exception
```

Comment:
While updating this file, a php-cs-fixer on it would have been nice to clean it up.